### PR TITLE
Data generation

### PR DESCRIPTION
**Pull request by Nguyephi**

### CODE REVIEW

File: data-generation.js [hunk 1]
```diff
@@ -0,0 +1,43 @@
...
+      avatar: faker.image.avatar(),
+      date: strDate,
+      stars: faker.random.number({ min: 0, max: 5 }),
+      review: faker.lorem.paragraph(),
+    };
+    records.push(reviewData);
+  }
+}
+
+csvWriter.writeRecords(records)
```
Comment (kraig-droid) [inline data-generation.js]: Ummm...it seems you are first generating all records then writing. This probably won't work for 10M records.

Comment (Nguyephi) [reply data-generation.js]: I was following the docs. What errors do you think I will come across?

Comment (Nguyephi) [reply data-generation.js]: There was an error haha. I will go back and fix it. I remember you posted something earlier today about this.

**Review by kraig-droid — COMMENTED** [review]
lgtm. Nice speed!

**Pull request merged**